### PR TITLE
Feature enable create flow on Goerli

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

_Provide a description of what this PR does. Link to any relevant GitHub issues, Notion tasks or Discord discussions._

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2341** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2341" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2341?utm_source=stack-comment).

File: src/pages/create/index.page.tsx [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-import { AppWrapper } from 'components/common'
+import { AppWrapper, Head } from 'components/common'
 import { DesmosScript } from 'components/common/Head/scripts/DesmosScript'
+import { Create } from 'components/Create'
 import { CV_V2, CV_V3 } from 'constants/cv'
 import { FEATURE_FLAGS } from 'constants/featureFlags'
 import { ThemeContext } from 'contexts/themeContext'
 import useMobile from 'hooks/Mobile'
-import Head from 'next/head'
+import NextHead from 'next/head'
```
Comment (tomquirk) [inline src/pages/create/index.page.tsx:10]: whats this do?

**Review by tomquirk — APPROVED** [review]
send it!

**Comment by wraeth-eth** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2341?utm_source=stack-comment).


Job Reference: 85wgSlSoKEYO8rUIoLww

**Pull request merged**